lib/init.coffee

```diff
@@ -10,4 +13,32 @@ module.exports =
       type: 'string'
 
   activate: ->
-    console.log 'activate linter-puppet-lint'
+    @regex = '(?<type>(warning|error)): (?<message>.+?) on line (?<line>\\d+) col (?<col>\\d+)'
```

Comment:
We are trying to drop `XRegExp` in favor of native ones, so could yuo try to port this to native regex?